openreview/conference/builder.py

```diff
@@ -109,6 +134,9 @@ def get_homepage_options(self):
             options['deadline'] = self.header.get('deadline')
         return options
 
+    def get_submissions(self, details = None):
```

Comment:
should this be get_blind_submissions()? If used on a single-blind, it will fail.

Reply:
it gets all submissions not matter the conference configuration. The trick is in get_blind_invitation_id.